packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Create an API token
+excerpt: How to create a token for the Okta API
+layout: Guides
+---
+
+This page explains what an API token is, why you need one, and how to create one.
+
```

Comment:
Add a `---` above and below the nutrition facts section, to render a horizontal line abover and below it.